src/constants/constants.js

```diff
@@ -439,7 +439,21 @@ export const OPERATIONS = {
     GET: 'get',
 };
 
-export const SERVICE_AGREEMENT_START_TIME_DELAY_FOR_COMMITS_SECONDS = 5 * 60;
+export const SERVICE_AGREEMENT_START_TIME_DELAY_FOR_COMMITS_SECONDS = {
+    mainnet: 5 * 60,
+    testnet: 5 * 60,
+    devnet: 3 * 60,
+    test: 10,
+    development: 10,
+};
+
+export const KNOWN_TRANSACTION_ERRORS = {
```

Comment:
Just a thought, maybe we can call it `handled_transaction_errors` or `expected_transaction_errors`